.github/workflows/main.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       matrix:
         node:
           - 'lts/hydrogen'
+          - '20'
```

Comment:
can we use the codename instead. I believe v20's codename is 'Iron'

Reply:
Nice, will update thank you